### PR TITLE
Feature login createaccount page

### PR DESCRIPTION
**Pull request by JirawadeeSampusri**

### CODE REVIEW

File: Kvent/views.py [hunk 1]
```diff
@@ -1,5 +1,54 @@
...
+            return HttpResponse("user Nooo")
+    
+
+def display_login(request):
+    return render(request, 'registration/login.html')
+
+def display_createaccount(request):
+    return render(request, 'registration/createaccount.html')
+
+def createaccount(request):
```
Comment (forfeen) [inline Kvent/views.py]: You should name the function by using underscore, _create_account_ .

File: Kvent/views.py [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+            else:
+                return HttpResponse("pass Nooo")
+        except User.DoesNotExist:
+            return HttpResponse("user Nooo")
+    
+
+def display_login(request):
+    return render(request, 'registration/login.html')
+
+def display_createaccount(request):
```
Comment (forfeen) [inline Kvent/views.py]: _'display_create_account'_

**Review by bleachjade — COMMENTED** [review]
you should not commit your __pycache__ onto github

File: mysite/urls.py [hunk 3]
```diff
@@ -18,5 +17,5 @@
 
 urlpatterns = [
     path('kvent/', include('Kvent.urls')),
-    path('admin/', admin.site.urls),
-]
+    path('admin/', admin.site.urls)
+]
```
Comment (bleachjade) [inline mysite/urls.py:25]: You should add all of the auth paths here:
`path('accounts/', include('django.contrib.auth.urls')),`

File: Kvent/views.py [hunk 4]
```diff
@@ -1,5 +1,54 @@
...
+            if user.password == password:
+                return HttpResponse("Yess")
+            else:
+                return HttpResponse("pass Nooo")
+        except User.DoesNotExist:
+            return HttpResponse("user Nooo")
+    
+
+def display_login(request):
+    return render(request, 'registration/login.html')
```
Comment (bleachjade) [inline Kvent/views.py]: Actually, I think there is no need to write these two function, you just use `{{next}}` in the login template file to redirect it to your return URL

File: Kvent/urls.py [hunk 5]
```diff
@@ -2,6 +2,12 @@
 
 from . import views
 
+app_name = "kvent"
 urlpatterns = [
     path('', views.index, name='index'),
+    path('accounts/createaccount', views.createaccount, name='createaccount'),
+    path('accounts/login', views.login, name='login'),
```
Comment (bleachjade) [inline Kvent/urls.py]: This is already included in `path('accounts/', include('django.contrib.auth.urls')),`, you don't have to declare it here.

**Pull request closed without merging**